### PR TITLE
Add proposal for Topics in OpenRTB.

### PR DESCRIPTION
**Pull request by timphsieh**

### CODE REVIEW

File: proposals/chrome-topics-api/openrtb_proposal.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            "id": "111,tax_v1,class_v1"
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+- Should the taxonomy version be defined as a separate extension to avoid repetitive text? This proposal is similar to [Segment Taxonomies](https://github.com/InteractiveAdvertisingBureau/openrtb/blob/master/extensions/community_extensions/segtax.md). For example:
```
Comment (sbelov) [inline proposals/chrome-topics-api/openrtb_proposal.md]: nit: maybe s/avoid repetitive text/reduce wire size?

Comment (sbelov) [inline proposals/chrome-topics-api/openrtb_proposal.md]: nit: maybe "Data object-level extension"

Comment (timphsieh) [reply proposals/chrome-topics-api/openrtb_proposal.md]: Sure, done.

Comment (timphsieh) [reply proposals/chrome-topics-api/openrtb_proposal.md]: Done.

**Review by sbelov — APPROVED** [review]

**Pull request merged**